docs/project/test-and-docs.md

```diff
@@ -184,27 +185,31 @@ terminal on Windows.
 
 		$ cd /c/gopath/src/github.com/docker/docker
     
-3. Set `DOCKER_CLIENTONLY` as follows:
+3. Set `DOCKER_REMOTE_DAEMON` as follows:
+
+		$ export DOCKER_REMOTE_DAEMON=1
 
-		$ export DOCKER_CLIENTONLY=1
-     
-	This ensures you are building only the client binary instead of both the
-	binary and the daemon.
-	
 4. Set `DOCKER_TEST_HOST` to the `tcp://IP_ADDRESS:2376` value; substitute your
-machine's actual IP address, for example:
+Linux machines actual IP address. For example:
 
 		$ export DOCKER_TEST_HOST=tcp://263.124.23.200:2376
```

Comment:
@calavera are you seeing `cannot to the docker daemon....` messages?
Try with `--ip-verify=false`